### DISCUSSION TITLE
when adding new pages to file as cropped boxes from one page i get all pages of the same look

### DISCUSSION THREAD

**Discussion by Swartor** [py-pdf/pypdf#1085] [Q&A]
I hav code like so and i get in new pdf file 3 pages but all looks the same as the last croped box. Why i cant get 3 difrent pages ??

```python
from PyPDF2 import PdfFileWriter, PdfFileReader

output = PdfFileWriter()
input1 = PdfFileReader(open("input.pdf", "rb"))
page = input1.getPage(0)
for y in range(0,3):
	page.cropBox.lowerRight = (205+185, 612)
	page.cropBox.upperLeft = (20+185, 752)
	output.addPage(page)
	
outputStream = open("output.pdf", "wb")
output.write(outputStream)
outputStream.close()
```

**Comment by JimInCO** [comment]
For anybody having this issue in the future I found a [SO response](https://stackoverflow.com/a/52344418) that explains how to get multiple pages. Basically, you need the copy module in order to make copies of the page object.
